packages/hash/frontend/src/blocks/page/createPlaceholderPlugin/Placeholder.tsx

```diff
@@ -0,0 +1,28 @@
+import { styled, Typography } from "@mui/material";
+
+const Command = styled(Typography)(({ theme }) => ({
+  color: theme.palette.gray[60],
+  borderRadius: 3,
+  background: theme.palette.gray[10],
+  width: 30,
+  height: 30,
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+}));
+
+const Container = styled(Typography)(({ theme }) => ({
+  position: "absolute",
+  display: "flex",
+  color: theme.palette.gray[60],
+  lineHeight: "27px",
```

Comment:
Why 27? Can we make this relative to the paragraph styling somehow, rather than have it disconnected from whatever styling is causing it to need to be 27?

Reply:
Should be solved here https://github.com/hashintel/hash/pull/953/commits/43e136eef1de516c16cd8c0d818b0d0434dd024d

Reply:
Apologies I didn't mention earlier, we have a design system that contains all the font sizes/line-heights we use. [internal link](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-App?node-id=780%3A111624). In this scenario, you can just give the Typography a variant of "regularTextPages" and it will apply the right line height

Reply:
@teenoh Thanks for the heads up, used the variant in this commit https://github.com/hashintel/hash/pull/953/commits/d83731125413398986be3bf9163506a246bc0e6b.